hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/optimize/TraversalUtil.java

```diff
@@ -571,6 +572,9 @@ public static void convAllHasSteps(Traversal.Admin<?, ?> traversal) {
         List<HasStep> steps =
                       TraversalHelper.getStepsOfAssignableClassRecursively(
                       HasStep.class, traversal);
+        if (steps.isEmpty() || !traversal.getGraph().isPresent()) {
+            return;
```

Comment:
can we add some comments for when the graph is not present  and why can return directly